src/GraphQL.Tests/Types/AllScalarGraphTypeTests.cs

```diff
@@ -622,6 +704,10 @@ public void parseValue_other_fail(Type graphType, object value)
     [InlineData(typeof(LongGraphType), 1.5)]
     [InlineData(typeof(ULongGraphType), 1.5)]
     [InlineData(typeof(BigIntGraphType), 1.5)]
+#if NET5_0_OR_GREATER
+    [InlineData(typeof(HalfGraphType), 1000000)]
+    [InlineData(typeof(HalfGraphType), -1000000)]
+#endif
```

Comment:
```suggestion
#if NET5_0_OR_GREATER
    [InlineData(typeof(HalfGraphType), false)]
#endif
```
the "other_fail" tests are for testing invalid types passed to the parseliteral method; out of range values are handled by parseLiteral_out_of_range